mako_rollup/src/main.rs

```diff
@@ -18,7 +29,24 @@ fn main() {
 
         // We only care about files, so we intentionally do nothing with directories
         if metadata.file_type().is_file() {
-            let account_uuid = &entry.path().parent().unwrap().to_str().unwrap()[7..];
+            let account_uuid;
+            let mut components = entry.path().components();
+            let next;
+
+            assert_eq!(components.next(), Some(Component::RootDir));
```

Comment:
I'm not sure how useful these asserts really are, given what you pass in to WalkDir::new().

If you do drop these, you can just do:

let component = componenets.skip(2).next().unwrap()

But if you keep them, this could be shorter as components.next().unwrap();

Reply:
I like the `let component = components.next().unwrap();` idea. Thanks!